### PR TITLE
tidyr blog post

### PR DESCRIPTION
**Pull request by hadley**

### CODE REVIEW

File: content/articles/2018-02-tidyr-0-8-0.Rmd [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+library(tibble)
+```
+
+We are pleased to announce that tidyr 0.8.0 is now available on CRAN. tidyr makes it easy to “tidy” your data, storing it in a consistent form so that it’s easy to manipulate, visualise and model. Tidy data has a simple convention: put variables in the columns and observations in the rows. You can learn more about it in the [tidy data vignette](http://tidyr.tidyverse.org/articles/tidy-data.html). Install it with:
+
+```{r, eval = FALSE}
+install.packages("tidyr")
+```
+
+This release mainly contains a bumper crop of small bug fixes and minor improvements, and a considerably increase in test coverage (84% to 99%). For the full details, see the [release notes](https://github.com/tidyverse/tidyr/releases/tag/v0.8.0). Here we'll highlight an important bug fix that might change existing code, and one new feature you might like to try out.
```
Comment (batpigandme) [inline content/articles/2018-02-tidyr-0-8-0.Rmd]: Swap one of the "mights" to a could, or just rephrase the last part of the sentence "…and a new feature to try out." (Hey, if they don't want to, I'm sure they won't!) 😉

File: content/articles/2018-02-tidyr-0-8-0.Rmd [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## API changes
+
+There was a bug in `separate()` where negative values had an off by one error. Now -1 correctly refers to the first position between characters counting from the right hand side.
```
Comment (batpigandme) [inline content/articles/2018-02-tidyr-0-8-0.Rmd]: `off-by-one error` (good ol' compound modifier strikes again...even wikipedia agrees with me here: https://en.wikipedia.org/wiki/Off-by-one_error). RHS is always dealers choice (righthand, right-hand, and right hand are all used interchangeably).

File: content/articles/2018-02-tidyr-0-8-0.Rmd [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+```{r}
+df <- tibble(x = c("male1", "female2", "male2"))
+df %>% separate(x, c("gender", "number"), -1)
+```
+
+## New features
+
+Thanks to the suggestion of [Andrew Bray](https://andrewpbray.github.io), tidyr can now "uncount" a data frame, duplicating aggregate rows
```
Comment (batpigandme) [inline content/articles/2018-02-tidyr-0-8-0.Rmd]: Colon or period at end of this sentence.

**Comment by hadley** [comment]
Thanks for the review!

**Pull request merged**